aries_vcx/src/handlers/issuance/holder.rs

```diff
@@ -52,21 +52,25 @@ impl Holder {
         Ok(())
     }
 
-    pub async fn send_request(
+    // todo: is the my_pw_did really necessary? is it used under the hood?
```

Comment:
this is something that has bugged me before about the holder API and older versions of credx/indy-sdk in general... i believe this gets attached in the credential request for it to act as a form of nonce for the issuer to sign over... however i don't believe it technically HAS to be a DID. 

In the past when i looked into this, deeper ursa code for issuer logic just refers to this value as a string called "prover id", rather than "prover did". unfortunately i believe libs like credx and vdrtools will enforce that this value MUST be a DID (when really it could be any arbitrary DID).

anoncreds-rs does a better job IMO by allowing an "entropy" string to be used instead when constructing a cred req; https://github.com/hyperledger/anoncreds-rs/blob/main/src/services/prover.rs#L107C34-L107C34

Reply:
Seems like we should create issue for this :-) It's really annoying indeed

Reply:
Perhaps remove this comment and add a link to the issue somewhere near the definition of `prover_create_credential_req`?

Reply:
✅